tests/testlib/nist_rng.c

```diff
@@ -0,0 +1,218 @@
+//  rng.c
```

Comment:
s2n's own RNG is already using AES_CTR_DRBG ... do we really need all of this?

Reply:
After adding AES256 and the no prediction resistance nope, got rid of all of these files.